getting-started/mag2exp.ipynb

```diff
@@ -0,0 +1,470 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import discretisedfield as df\n",
+    "import numpy as np\n",
+    "import micromagneticmodel as mm\n",
+    "\n",
+    "np.random.seed(2)\n",
+    "\n",
+    "mesh = df.Mesh(\n",
+    "    p1=(-200e-9, -200e-9, -20e-9),\n",
+    "    p2=(200e-9, 200e-9, 20e-9),\n",
+    "    cell=(8e-9, 8e-9, 10e-9),\n",
+    ")\n",
+    "\n",
+    "system = mm.System(name='mag2exp_test')\n",
+    "\n",
+    "# Values for MnSi\n",
+    "system.energy = (\n",
+    "    mm.Exchange(A=15e-12)\n",
+    "    + mm.DMI(D=3e-3, crystalclass=\"T\")\n",
+    "    + mm.UniaxialAnisotropy(K=0.2e6, u=(0, 0, 1))\n",
+    "    + mm.Zeeman(H=(0, 0, 3.5e5))\n",
+    "    + mm.Demag()\n",
+    ")\n",
+    "\n",
+    "Ms = 580e3\n",
+    "\n",
+    "def m_fun(pos):\n",
+    "    return 2 * np.random.rand(3) - 1\n",
+    "\n",
+    "# create system with above geometry and initial magnetisation\n",
+    "system.m = df.Field(mesh, nvdim=3, value=m_fun, norm=Ms)\n",
+    "\n",
+    "system.m.sel(\"z\").mpl()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Relax the system"
```

Comment:
Maybe you can add one sentence after minimising and plotting the magnetisation to roughly say what we see. I guess some sort of Bloch skyrmion lattice?